src/Symfony/Component/Serializer/Exception/AggregatedException.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Exception;
+
+final class AggregatedException extends RuntimeException implements AggregableExceptionInterface
+{
+    /**
+     * @var bool
+     */
+    public $has = false;
```

Comment:
I will be picky sorry :)
It's weird to me to have simply `has` what about `isEmpty` ?
In the [conventions](https://symfony.com/doc/current/contributing/code/conventions.html) the method `has()` will be used to check if an item exists so even if in this case for performance reason it's a property I think we should try to be consistent.
```suggestion
    public $isEmpty = true;
```

Reply:
done